common/ECSResourcesCheck/src/main/java/Main.java

```diff
@@ -0,0 +1,14 @@
+import org.json.JSONObject;
+
+public class Main {
+    public static JSONObject readParam(String param){
+        return new JSONObject(param);
+    }
+
+    public static void main(String[] args){
```

Comment:
Please document the variables this function expects according to their position. For example:
```suggestion
        /*
        *  This is the main method where the program starts.
        *   args[0] = x, args[1] = y, args[2] = z ...
        */
        public static void main(String[] args){
```

Reply:
done